packages/testcontainers/src/containers/RegTestContainer/Masternode.ts

```diff
@@ -229,15 +229,15 @@ export class MasterNodeRegTestContainer extends RegTestContainer {
    * Wait for active price
    *
    * @param {string} fixedIntervalPriceId
-   * @param {number} activePrice
+   * @param {string} activePrice
    * @param {number} [timeout=30000] in ms
    * @return {Promise<void>}
    */
-  async waitForActivePrice (fixedIntervalPriceId: string, activePrice: number, timeout = 100000): Promise<void> {
+  async waitForActivePrice (fixedIntervalPriceId: string, activePrice: string, timeout = 30000): Promise<void> {
     return await waitForCondition(async () => {
       const data: any = await this.call('getfixedintervalprice', [fixedIntervalPriceId])
       // eslint-disable-next-line
-      if (data.activePrice !== activePrice) {
+      if (data.activePrice.toString() !== activePrice) {
         await this.generate(1)
         return false
       } else {
```

Comment:
style this way
```
if (....) {
  ....
  return false
}
return true

Reply:
Ya your way is recommended by one of the linter. I forget the name